client/components/Menu/Menu.jsx

```diff
@@ -91,11 +69,11 @@ class ResponsiveDrawer extends React.Component {
           {['Forum', 'Creators', 'Contact', 'Setting'].map((text, index) => (
```

Comment:
Don't think this line needs index either does it?

Reply:
No, it shouldn't! The index parameter was used for the matching the material-ui icons. Should've taken that off, my bad!